### PR TITLE
Add an important code to add when customizing a ContentDialog

### PR DESCRIPTION
**Pull request by ArchieCoder**
GitHub Issue (If applicable): closes #

## PR Type

What kind of change does this PR introduce?

## PR Checklist

Please check if your PR fulfills the following requirements:

- [ ] Docs have been added/updated which fit [documentation template](https://github.com/unoplatform/uno/blob/master/doc/.feature-template.md) (for bug fixes / features)
- [ ] [Unit Tests and/or UI Tests](https://github.com/unoplatform/uno/blob/master/doc/articles/uno-development/working-with-the-samples-apps.md) for the changes have been added (for bug fixes / features) (if applicable)
- [ ] Validated PR `Screenshots Compare Test Run` results.
- [ ] Contains **NO** breaking changes
- [ ] Associated with an issue (GitHub or internal) and uses the [automatic close keywords](https://help.github.com/en/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue).
- [ ] Commits must be following the [Conventional Commits](https://www.conventionalcommits.org/en/v1.0.0/#summary) specification.

## Other information

Internal Issue (If applicable):

### CODE REVIEW

File: doc/articles/features/dialogs.md [hunk 1]
```diff
@@ -30,6 +30,21 @@ ContentDialogResult result = await noWifiDialog.ShowAsync();
...
+## Customize a `ContentDialog`
+
+You can use various properties to customize its display and also provide a custom XAML-based content for it. For more information, see [ContentDialog Class | Microsoft Learn](https://learn.microsoft.com/windows/windows-app-sdk/api/winrt/microsoft.ui.xaml.controls.contentdialog).
+
+In order for your custom dialog to follow the Fluent design, it's important that you add the following line in the constructor of your dialog:
+
+```csharp
+Style = Application.Current.Resources["DefaultContentDialogStyle"] as Style;
+```
+You dialog will look similar to this dialog:
```
Comment (agneszitte) [inline doc/articles/features/dialogs.md:42]:
```suggestion
Your dialog will look similar to this dialog:
```

**Review by agneszitte — COMMENTED** [review]
@ArchieCoder thanks a lot for your contribution!

Just a few details for the checks to pass for the pull request:
- [ ] You will need to adjust your previous commits, they must follow the [Conventional Commits](https://www.conventionalcommits.org/en/v1.0.0/#summary) specification.
      Like `docs: Add customizing details for ContentDialog` for example
- [ ] Also, you will need to fix the markdown linting errors please: https://dev.azure.com/uno-platform/Uno%20Platform/_build/results?buildId=126777&view=logs&jobId=e61e0ee1-f3da-5962-52cf-48bdde3a5bd4&j=e61e0ee1-f3da-5962-52cf-48bdde3a5bd4&t=c18ecb7c-3c97-55c3-9bb9-5ef1777cbc25

image

FYI you can use github.dev web-based editor to adjust your changes if needed:
You can use https://docs.github.com/en/codespaces/the-githubdev-web-based-editor#opening-the-githubdev-editor

File: doc/articles/features/dialogs.md [hunk 2]
```diff
@@ -12,7 +12,7 @@ uid: Uno.Features.Dialogs
 
 ## Using `ContentDialog`
 
-The recommended way to display user dialogs is via the `Microsoft.UI.Xaml.Controls.ContentDialog` class. You can use various properties to customize its display and also provide a custom XAML-based content for it. For more information, see [ContentDialog Class | Microsoft Learn](https://learn.microsoft.com/windows/windows-app-sdk/api/winrt/microsoft.ui.xaml.controls.contentdialog).
+The recommended way to display user dialogs is via the `Microsoft.UI.Xaml.Controls.ContentDialog` class.
```
Comment (agneszitte) [inline doc/articles/features/dialogs.md:15]:
```suggestion
The recommended way to display user dialogs is via the `Microsoft.UI.Xaml.Controls.ContentDialog` class.
```

File: doc/articles/features/dialogs.md [hunk 3]
```diff
@@ -30,6 +30,21 @@ ContentDialogResult result = await noWifiDialog.ShowAsync();
 
 It is crucial to set the `XamlRoot` property before calling `ShowAsync`. This way the dialog will become asssociated with the visual tree. `XamlRoot` can be retrieved from any loaded control in your window (e.g. a `Button`, your `Page`, etc.).
```
Comment (agneszitte) [inline doc/articles/features/dialogs.md:31]:
```suggestion
It is crucial to set the `XamlRoot` property before calling `ShowAsync`. This way the dialog will become associated with the visual tree. `XamlRoot` can be retrieved from any loaded control in your window (e.g. a `Button`, your `Page`, etc.).
```

File: doc/articles/features/dialogs.md [hunk 4]
```diff
@@ -30,6 +30,21 @@ ContentDialogResult result = await noWifiDialog.ShowAsync();
...
+```
+You dialog will look similar to this dialog:
+<img width="273" alt="WithStyle" src="https://github.com/unoplatform/uno/assets/1608424/ca1c3dd8-694e-4149-8778-cbf103098fd3">
+
+If you omit this line, your dialog will look like this:
+<img width="281" alt="NoStyle" src="https://github.com/unoplatform/uno/assets/1608424/e1066618-1d6b-4215-8156-f1cf21a30c9d">
+
 ## Using `MessageDialog`
 
 `MessageDialog` is a legacy API which is no longer officially supported. If possible, please use `ContentDialog` instead.
```
Comment (agneszitte) [inline doc/articles/features/dialogs.md:50]:
```suggestion
`MessageDialog` is a legacy API that is no longer officially supported. If possible, please use `ContentDialog` instead.
```

File: doc/articles/features/dialogs.md [hunk 5]
```diff
@@ -30,6 +30,21 @@ ContentDialogResult result = await noWifiDialog.ShowAsync();
...
+In order for your custom dialog to follow the Fluent design, it's important that you add the following line in the constructor of your dialog:
+
+```csharp
+Style = Application.Current.Resources["DefaultContentDialogStyle"] as Style;
+```
+You dialog will look similar to this dialog:
+<img width="273" alt="WithStyle" src="https://github.com/unoplatform/uno/assets/1608424/ca1c3dd8-694e-4149-8778-cbf103098fd3">
+
+If you omit this line, your dialog will look like this:
+<img width="281" alt="NoStyle" src="https://github.com/unoplatform/uno/assets/1608424/e1066618-1d6b-4215-8156-f1cf21a30c9d">
```
Comment (agneszitte) [inline doc/articles/features/dialogs.md:46]: @ArchieCoder the images would need to be placed in the assets folder for them to display properly on the documentation website please
Here is an example: https://github.com/unoplatform/uno/blob/4fae8b53d71ad87ca0f84e0d603cda53c6bf205c/doc/articles/features/ElevatedView.md?plain=1#L37

**Review by MartinZikmund — APPROVED** [review]

**Comment by agneszitte** [comment]
Closing this PR in favor of https://github.com/unoplatform/uno/pull/16975

**Pull request closed without merging**